cluster/gh-action-runner/gh-action-runner-template-create.yml

```diff
@@ -0,0 +1,262 @@
+          auth: "{{ovirt_auth}}"
+          state: stopped
+
+      - ovirt_vm_info:
+          pattern: name=fedora_test_vm and cluster="{{ovirt_cluster_name}}"
+          auth: "{{ovirt_auth}}"
+          follow: ['disk_attachments']
+        register: result
+     
+      - debug:
+          msg: "{{ result  }}"
+
+      - name: Increase root disk size
+        ovirt_disk:
+          auth: "{{ ovirt_auth }}"
+          id: "{{ result.ovirt_vms[0].disk_attachments[0].disk.id }}"
+          size: "50GiB"
+          wait: true
+          vm_name: "fedora_test_vm"
+          bootable: true
+
+      - name: start the VM
+        ovirt_vm:
+          name: "fedora_test_vm"
+          cluster: "{{ovirt_cluster_name}}"
+          auth: "{{ovirt_auth}}"
+          state: running
+
+      - ovirt_vm_info:
+          pattern: name=fedora_test_vm and cluster="{{ovirt_cluster_name}}"
+          auth: "{{ovirt_auth}}"
+          follow: ['reported_devices']
+        register: result
+        until: "result.ovirt_vms | ovirt.ovirt.ovirtvmipv4 | length > 0"
+        retries: 10
+        delay: 30
+
+      - set_fact:
+          collected_address: "{{ result.ovirt_vms | ovirt.ovirt.ovirtvmipv4 }}"
+    
+      - debug: msg="got VM IP {{collected_address}}"
+
+      - blockinfile:
+          path: /tmp/ssh-conf
+          create: yes
+          mode: 600
+          content: |
+            Host ovirt-proxy-vm-1.rhv45.gcp.devcluster.openshift.com
+              StrictHostKeyChecking no
+              UserKnownHostsFile /dev/null
+              IdentityFile /tmp/id_ssh_rsa
+
+            Host 192.168.225.*
+              ProxyJump installer@ovirt-proxy-vm-1.rhv45.gcp.devcluster.openshift.com
+              User root
+              StrictHostKeyChecking no
+              UserKnownHostsFile /dev/null
+              IdentityFile /tmp/id_ssh_rsa
+        tags:
+          - always          
+
+      - name: remove test-vm VM from the engine
+        ovirt_vm:
+          name: "fedora_test_vm"
+          state: absent
+          auth: "{{ovirt_auth}}"
+        tags:
+          - never
+          - cleanup  
+
+      - debug:
+          msg: "{{ result.ovirt_vms[0].id }}"
+
+      - shell: "echo {{ result.ovirt_vms[0].id }} >/tmp/test_output/test_vm_id"
+
+      - debug: var=output.stdout_lines                
+
+      - name: Add a host alias that we reach through a tunnel
+        add_host:
+          hostname: '{{ collected_address }}'
+          name: test-vm
+          ansible_ssh_host: '{{ collected_address }}'
+          ansible_ssh_user: root    
+          ansible_ssh_common_args: '-F /tmp/ssh-conf'      
+        tags:
+          - always         
+      - name: check connectivity to the VM
+        ping:
+        delegate_to: test-vm
+
+      - name: resize the root partition 
+        shell: |
+          growpart  /dev/vda 1
+          resize2fs /dev/vda1
+        delegate_to: test-vm
+
+      - name: upgrade to fedora-36
```

Comment:
Why do we need to upgrade to fedora 36?
What is the base OS version and why can't we install the 36 originally?